proposals/ard_surface_reflectance.json

```diff
@@ -0,0 +1,150 @@
+{
+    "id": "ard_surface_reflectance",
+    "summary": "CARD4L compliant Surface Reflectance generation",
+    "description": "...",
+    "categories": [
+        "cubes",
+        "sar",
+        "ard"
+    ],
+    "experimental": true,
+    "parameters": [
+        {
+            "description": "The source data cube containing multi-spectral optical top of the atmosphere (TOA) reflectances.",
+            "name": "data",
+            "schema": {
+                "subtype": "raster-cube",
+                "type": "object"
+            }
+        },
+        {
+            "description": "The digital elevation model to use, leave empty to allow the back-end to make a suitable choice.",
+            "name": "elevation_model",
+            "optional": true,
+            "default": null,
+            "schema": [
+                {
+                    "type": "string",
+                    "subtype": "collection-id"
+                },
+                {
+                    "type": "null"
+                }
+            ]
+        },
+        {
+            "description": "The minimum probability required for atmospheric disturbances like clouds for being removed (greater-than-equal).",
+            "name": "threshold",
+            "optional": true,
+            "default": 0.5,
```

Comment:
For ARD, I would dare to go for a lower probability, maybe 0.3. Cloud pixels are very bad for analysis, and the general consensus seems to be that it's better to throw out more good pixels than keeping in clouded pixels. @sinergise-anze  What threshold value do you use to convert cloud probabilities into masks?

Reply:
Being more strict towards possible disturbances in the designated ARD processes also aligns well with the ARD Concept CEOS stated in their first webinar. Pixel values should represent physically comparable measurements across timesteps, space (and across sensors).

Reply:
Sounds good. Is 0.3 a "random" value or is there some science behind it? Or is it common denominator between a subset of the tools we use? Then we could maybe link to something and make it look less random (like it was with 0.5, too).

Reply:
I think the value depends a lot on the intended use of the mask. Since the process can't know what the caller expects, maybe it would be a good idea if this was a required parameter without a default value?

Reply:
But you have a CLM and CLP band right? I assume CLM is CLP with a threshold applied, or how else do you get it?

Reply:
This example mentions 0.4 as a default:
https://github.com/sentinel-hub/sentinel2-cloud-detector/blob/master/examples/sentinel2-cloud-detector-example.ipynb

Reply:
I agree that if there's no reasonable default then we should make the parameter required.

Reply:
> But you have a CLM and CLP band right? I assume CLM is CLP with a threshold applied, or how else do you get it?

Sorry, didn't understand what you meant - yes, CLM uses a predefined threshold value of `0.4`.

Reply:
We've removed probability completely for now as we got into discussions whether the probability actually has to be per "aspect" (e.g. cloud or shadows) separately. It seems that's more useful, but on the other hand, then we need an object that maps the probability values to the respective aspect and that is complicated. Thus, we decided, for now, to remove it completely and just use the defaults of the chosen method.